Source/WebCore/html/HTMLScriptElement.h

```diff
@@ -37,8 +40,17 @@ class HTMLScriptElement final : public HTMLElement, public ScriptElement {
 
     String text() const { return scriptContent(); }
     WEBCORE_EXPORT void setText(String&&);
+    ExceptionOr<void> setText(std::variant<RefPtr<TrustedScript>, String>&&);
+
+    ExceptionOr<void> setTextContent(std::optional<std::variant<RefPtr<TrustedScript>, String>>&&);
+
+    using Node::setTextContent;
+    using HTMLElement::setInnerText;
```

Comment:
It’s a bit strange to have this after our `setTextContent` function and before our `setInnerText` function. I suggest we put each one just above the function of the same name, in a shared paragraph:
```
using Node::setTextContent;
ExceptionOr<void> setTextContent(std::optional<std::variant<RefPtr<TrustedScript>, String>>&&);
```